unified-codes/tools/scripts/dgraph/data/v2/schema.gql

```diff
@@ -0,0 +1,83 @@
+type Property {
+  type
+  value
+}
+
+type Category {
+  code
+  name
+  children
+  properties
+}
+
+type Product {
+  code
+  name
+  combines
+  children
+  properties
+}
+
+type Route {
+  code
+  name
+  children
+  properties
+}
+
+type DoseForm {
+  code
+  name
+  children
+  properties
+}
+
+type DoseQualification {
+  code
+  name
+  children
+  properties
+}
+
+type DoseStrength {
+  code
+  name
+  children
+  properties
+}
+
+type DoseUnit {
+  code
+  name
+  children
+  properties
+}
+
+type InnerPack {
+  code
+  name
+  children
+  properties
+}
+
+type PackSize {
+  code
+  name
+  children
+  properties
+}
+
+type OuterPack {
+  code
+  name
+  children
+  properties
+}
+
+code: string @index(exact, fulltext).
+name: string @lang @index(exact, term, trigram) .
+type: string @index(term) .
```

Comment:
The type defined here is mainly for the `Property` Dgraph type (which currently only has `type` and `value` properties). I agree bit confusing to add to all the other types. Seems a bit redundant?

Reply:
Hmm, you're right actually. We will need to add to the `gql` mappers, but I don't think we need if for Dgraph at all (think we can get away with it for properties too, since we will have instances for each one)!